### PR TITLE
Artwork: Update Enforcer Riot Staff and Enforcer Confrontation Gear

### PR DESCRIPTION
**Pull request by 1010todd**
----------------------
**Content (Artwork)**

## Summary
Remade the Enforcer Riot Staff in the same style as Enforcer Confrontation Gear per [Amazinite's request](https://github.com/endless-sky/endless-sky/pull/6709#pullrequestreview-941940693) and recolored Enforcer Confrontation Gear to regular Coalition theme.

New Enforcer Riot Staff:

Coal EFS 15-nrwmid-newlight semiopen-2inv-9 10x

Recolored Enforcer Confrontation Gear:

Coal ECG altmat-8

Outfitter Mockup:
coal h2h compare

<details>
<summary>Blue theme</summary>

Coal EFS 15-nrwmid-newlight semiopen-1 10x

Outfitter Mockup:
coal h2h compare stf

Coal EFS 15-nrwmid-newlight semiopen-1 side 10x

Coal EFS 15-nrwmid-newlight semiopen-1 bg 10x

</details>

<details>
<summary>Alternative color scheme</summary>
(Will also remake Enforcer Confrontation Gear in this case.)

Coal EFS 15-nrwmid-newlight semiopen-1inv-5 10x

coal h2h compare alt staff

Enforcer Confrontation Gear with the same color scheme.
Coal ECG altmat-6

coal h2h compare

</details>

Note: I also edited a mistake in the copyright I made in the previous PR here, it's just one line so I bundled it with this. 

Note on exact way to replicate; Since I happen to have Krita 4.4.3 open I used it to rescaled the images from 5x render with  `Image > Scale image to new size` and use Lanczos3 filter.

## Save File
This save file can be used to play through the new mission content:
(Use some dev/cheat plugin and go there.)

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/endless-sky/endless-sky-assets/pull/106
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/327

### CODE REVIEW

**Comment by EricD112** [comment]
I quite like the updated staff, but it doesn't feel like part of the coalition with the current colour scheme. The alternative scheme looks pretty good in my opinion and makes the staff feel correct/fitting, so I'd prefer the alternative scheme.

File: copyright [hunk 1]
```diff
@@ -72,13 +72,13 @@ License: CC-BY-SA-4.0
...
 Files:
  images/ship/hai?flea*
+ images/projectile/rail?slug*
 Copyright: Matthew Smestad
 License: CC-BY-SA-4.0
 
 Files:
  images/outfit/railgun*
  images/outfit/railslug*
- images/projectile/rail?slug*
```
Comment (Hurleveur) [inline copyright:81]: why did you remove the railgun slug? seems unrelated

Comment (1010todd) [reply copyright:81]: Made a mistake in previous PR, I didn't really touched the projectile sprite.

**Comment by Hurleveur** [comment]
Honestly I prefer the alternative color scheme but do note I'm not artist

**Comment by rovermicrover** [comment]
The alternative color scheme is pretty awesome.

**Comment by McloughlinGuy** [comment]
I’m no art reviewer, but I like alternate color scheme a lot

**Comment by Saugia** [comment]
If you can I'd like to see the coalition color scheme version next to this as well that we saw in discord.

**Review by beccabunny — APPROVED** [review]
Gotta fix the copyright in the high DPI PR, but this one is good to go already. Well done

**Comment by Saugia** [comment]
Ditto. Fix the High DPI PR, and this is good to go.

**Comment by 1010todd** [comment]
Uploaded the assets.
https://github.com/endless-sky/endless-sky-assets/pull/106

**Review by Saugia — APPROVED** [review]
